docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.4"
+services:
+  orchard:
+    image: orchard
+    depends_on:
+      - postgres
+    ports:
+      - 80:80
+    networks:
+      - orchardnet
+    environment:
+      OrchardCore__OrchardCore.Setup.AutoSetup__SiteName: "AutoSetup Example"
+      OrchardCore__OrchardCore.Setup.AutoSetup__SiteTimeZone: "Europe/Amsterdam"
+      OrchardCore__OrchardCore.Setup.AutoSetup__AdminUsername: "Admin"
+      OrchardCore__OrchardCore.Setup.AutoSetup__AdminEmail: "info@orchardproject.net"
+      OrchardCore__OrchardCore.Setup.AutoSetup__AdminPassword: "OrchardCoreRules1!"
+      OrchardCore__OrchardCore.Setup.AutoSetup__DatabaseProvider: "Postgres"
+      OrchardCore__OrchardCore.Setup.AutoSetup__DatabaseName: "OrchardCore"
+      OrchardCore__OrchardCore.Setup.AutoSetup__DatabaseConnectionString: "User ID=postgres;Password=postgres;Host=postgres;Port=5432;Database=orchardcore;EntityAdminDatabase=postgres;Pooling=true;"
+      OrchardCore__OrchardCore.Setup.AutoSetup__DatabaseTablePrefix: "OrchardCore_"
+      OrchardCore__OrchardCore.Setup.AutoSetup__RecipeName: "Agency"
+      OrchardCore__OrchardCore.Setup.AutoSetup__CreateDatabase: "true"
+      Logging__LogLevel__Default: "Information"
+  postgres:
+    image: postgres:10-alpine
+    ports:
+      - 5432:5432
+    networks:
+      - orchardnet
+    volumes:
+      - type: volume
+        source: postgres_db
+        target: /var/lib/postgresql/data
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_INITDB_ARGS: "--data-checksums"
+networks:
+  orchardnet:
+volumes:
+  postgres_db:
```

Comment:
new line at the end of file

Reply:
Should we keep this file? Or can it be deleted?

Reply:
Yes, I think this should be deleted